src/views/teamList/TeamList.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {useEffect, useState} from 'react'
+import { Link } from 'react-router-dom';
+import { deleteTeamById, getTeams } from '../../services/teams';
+
+export const TeamList = () => {
+    const [teams, setTeams] = useState([]);
+    const [loading, setLoading] =useState(true)
+ 
+    const loadTeams = async () => {
+        const res = await getTeams()
+        setTeams(res)
+        setLoading(false)
+    }
+
+    useEffect(() => {loadTeams()},[])
+
+    const handleDelete = async ({id, name}) => {
```

Comment:
Deleting a team with players attached doesn't give a user error. You could give a user error that the user needs to delete the associated users first or add the player deletion to the team deletion handler. Either way will work